src/Service.Host/client/src/components/parcels/ParcelsSearch.js

```diff
@@ -0,0 +1,329 @@
+    };
+
+    return (
+        <Page>
+            <Grid spacing={3} container justify="center">
+                <Grid item xs={11} />
+                <Grid item xs={1}>
+                    {/* todo: add if allowedToCreate properly, still needs pulled in from backend */}
+                    {(allowedToCreate || true) && (
+                        <LinkButton text="Create" to="/inventory/parcels/create" />
+                    )}
+                </Grid>
+                <Grid item xs={1}>
+                    <SearchInputField
+                        label="Parcels"
+                        fullWidth
+                        placeholder="search.."
+                        onChange={handleParcelNumberSearchChange}
+                        propertyName="parcelSearchTerm"
+                        type="text"
+                        value={parcelNumberSearch}
+                        debounce={5000}
+                    />
+                </Grid>
+
+                <Grid item xs={2}>
+                    <Dropdown
+                        onChange={handleSupplierSearchChange}
+                        items={suppliers
+                            .map(s => ({
+                                ...s,
+                                id: s.id,
+                                displayText: `${s.name} ( ${s.id} )`
+                            }))
+                            .sort((a, b) => (a.id > b.id ? 1 : -1))}
+                        value={supplierIdSearch}
+                        propertyName="supplierIdSearch"
+                        required
+                        fullWidth
+                        label="Supplier"
+                        allowNoValue
+                    />
+                </Grid>
+
+                <Grid item xs={2}>
+                    <Dropdown
+                        onChange={handleCarrierSearchChange}
+                        items={carriers
+                            .map(s => ({
+                                ...s,
+                                id: s.carrierCode,
+                                displayText: `${s.carrierCode} ( ${s.organisationId} )`
+                            }))
+                            .sort((a, b) => (a.id > b.id ? 1 : -1))}
+                        value={carrierIdSearch}
+                        propertyName="carrierIdSearch"
+                        required
+                        fullWidth
+                        label="Carrier"
+                        allowNoValue
+                    />
+                </Grid>
+
+                <Grid item xs={2}>
+                    <SearchInputField
+                        label="Date Created"
+                        fullWidth
+                        placeholder="search.."
+                        onChange={handleDateSearchChange}
+                        propertyName="dateCreatedSearchTerm"
+                        type="date"
+                        value={dateCreatedSearch}
+                        debounce={5000}
```

Comment:
long debounce?

Reply:
I don't think these even take a debounce param, will take it out